src/tags_int.cpp

```diff
@@ -3265,8 +3265,8 @@ namespace Exiv2 {
         }
 
         std::string stringValue = value.toString();
-        if (stringValue.at(19) == 'Z') {
-            stringValue = stringValue.substr(0, 19);
```

Comment:
Unneeded creation of a _new_ string, which involves copying and memory allocation.